cmd/generator.go

```diff
@@ -25,6 +25,7 @@ var (
 	queryInterval          = kingpin.Flag("query-interval", "Frequency to query each tenant.").Default("10s").Duration()
 	queryTimeout           = kingpin.Flag("query-timeout", "Query timeout.").Default("30s").Duration()
 	queryMaxAge            = kingpin.Flag("query-max-age", "How back in the past metrics can be queried at most.").Default("24h").Duration()
+	additionalQueries      = kingpin.Flag("additional-queries", "PromQL queries to run in addition to the default.").Strings()
```

Comment:
Note to reviewers: wasn't sure whether to follow the `query-...` naming pattern or not, open to feedback.

Reply:
I think would be a nice to have, but not a blocker.

Reply:
Changed in fc8f80a